app/models/batch.rb

```diff
@@ -56,13 +57,8 @@ class Batch < ApplicationRecord # rubocop:todo Metrics/ClassLength
   accepts_nested_attributes_for :requests
   broadcast_with_warren
 
-  validate :requests_have_same_read_length,
-           :requests_have_same_flowcell_type,
-           :batch_meets_minimum_size,
-           :all_requests_are_ready?,
-           :requests_have_same_target_purpose,
-           on: :create,
-           if: :pipeline
+  # Validations for batches
+  validates_with BatchValidator, on: :create, if: :pipeline
```

Comment:
A better name might be BatchCreationValidator if this validation only runs on create.